src/hiero_sdk_python/transaction/transaction_record.py

```
@@ -73,6 +73,7 @@ class TransactionRecord:
 
     prng_number: Optional[int] = None
     prng_bytes: Optional[bytes] = None
+    duplicates: List['TransactionRecord'] = field(default_factory=list)
```

Ruff (0.14.14)

[error] Undefined name `List`

(F821)